sonatype.sbt

```diff
@@ -1,12 +1,3 @@
-ThisBuild / publishTo := sonatypePublishToBundle.value
-
-ThisBuild / scmInfo := Some(ScmInfo(
-  url("https://github.com/guardian/mobile-apps-api-models"),
-  "scm:git:git@github.com:guardian/mobile-apps-api-models.git"
-))
-
-ThisBuild / homepage := Some(url("https://github.com/guardian/mobile-apps-api-models"))
-
 ThisBuild / developers := List(Developer(
```

Comment:
TODO: I'm not sure if this is needed, or really what value this provides

Reply:
Yep, you can delete this entire `sonatype.sbt` file!

I need to add information on this to the [`gha-scala-library-release-workflow`](https://github.com/guardian/gha-scala-library-release-workflow) readme to make this clearer, but all the sbt settings in this `sonatype.sbt` file are automatically set by `gha-scala-library-release-workflow`, so the entire file can be removed. See also https://github.com/guardian/facia-scala-client/pull/299 as a commented example of removing lots of fields.